examples/dataclasses-avroschema-example/dataclasses_avroschema_example/app.py

```diff
@@ -0,0 +1,53 @@
+from . import serializers
+from .models import Address, User
+from kstreams import create_engine, Stream
+
+import asyncio
+
+user_topic = "local--avro-user"
+address_topic = "local--avro-address"
+
+stream_engine = create_engine(
+    title="my-stream-engine",
+    value_serializer=serializers.AvroSerializer(),
+)
+
+
+@stream_engine.stream(
+    user_topic, value_deserializer=serializers.AvroDeserializer(model=User)
+)
+async def user_stream(stream: Stream):
+    async for cr in stream:
+        print(f"Event consumed on topic {user_topic}. The user is {cr.value}")
```

Comment:
maybe also print the type of `cr.value`:
```python
print(f"Event consumed on topic {user_topic}. The user is {cr.value}. User type: {type(cr.value)}")
```

Reply:
it is not necessary as `cr.value` is an instance and it has the `__repr__()` method. It is clear in the `logs` to see what is what.